service/src/main/java/org/ehrbase/service/EhrServiceImp.java

```diff
@@ -136,11 +136,12 @@ public EhrResult updateStatus(
 
         // pre-step: validate + check for valid ehrId
         check(status);
-        checkEhrExists(ehrId);
         checkEhrExistForParty(ehrId, status);
 
         UUID ehrStatusId = UUID.fromString(ifMatch.getObjectId().getValue());
-        int version = Integer.parseInt(ifMatch.getVersionTreeId().getValue());
+        int version = ifMatch.hasExtension()
+                ? Integer.parseInt(ifMatch.getVersionTreeId().getValue())
+                : 0;
```

Comment:
@HolgerReiseVSys I'm not entirely sure here - but should we reject If-Match header that contain an Object-Version-IDs without a proper system and/or version extension?

Reply:
will open a new ticket for enhancuing VoId handling.